### DISCUSSION TITLE
Graphical map could show Fief Workflow

### DISCUSSION THREAD

**Discussion by ClaudioBsh** [fief-dev/fief#331] [Ideas]
It would be nice to have - in the official documentation - a map, a graphical overview how workspaces, tenants and clients are working together for different situations like one server with one web application to multiple servers/domains with multi web applications. 

"One picture shows more then thousand words":-)

**Comment by frankie567** [comment]
Sure, good idea!

Even though, as I mentioned in your previous thread, we'll remove the multiple workspaces architecture ; so things will (hopefully) much easier to reason with 🙂
